EventView-1/src/main/java/com/eventview/repo/EventTypeRepoImpl.java

```diff
@@ -67,8 +67,15 @@ public boolean eventTypeExists(Integer eventTypeId) {
     }
 
     @Override
-    public String getEventType(Integer eventId) {
-        String SELECT_EVENT_TYPE_BY_EVENT = "SELECT event_type FROM eventtypes WHERE event_type_id = (SELECT event_type_id FROM events WHERE events.event_id = ?)";
-        return jdbcTemplate.queryForObject(SELECT_EVENT_TYPE_BY_EVENT, new Object[]{eventId}, String.class);
+    public List<String> getEventTypeToday() {
+        String SELECT_EVENT_TYPE_TODAY = "select event_type from eventtypes where event_type_id in (select event_type_id from events where day(now()) = day(event_date))";
+        return jdbcTemplate.queryForList(SELECT_EVENT_TYPE_TODAY, String.class);
+    }
+
+
+    @Override
+    public List<String> getEventTypeWeek() {
+        String SELECT_EVENT_TYPE_WEEK = "select event_type from eventtypes where event_type_id in (select event_type_id from events where day(event_date) BETWEEN day(now()) and day(NOW()) + 7)";
+        return jdbcTemplate.queryForList(SELECT_EVENT_TYPE_WEEK,String.class);
```

Comment:
7 shd be coming from property also why cant join this with user and eventtype table and get all needed info in single query instead all drama u doing

Reply:
okay i'll fix